src/averageAges.js

```diff
@@ -15,7 +15,10 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const men = people.filter(person => person.sex === 'm'
```

Comment:
use destructure assignment throughout your code
```suggestion
  const men = people.filter(({ sex })=> sex === 'm'
```